backend/data_handler_app/models.py

```diff
@@ -22,41 +22,29 @@ class Answer(models.Model):
     def __str__(self):
         return self.answer
 
-class ClientData(models.Model):
-    # id field is automatically generated by django, we
-    # can create seperate id's for daily visitors, monthly
-    # visitors, etc.
-    date = models.DateTimeField(auto_now_add=True)
-    start_time = models.TimeField(auto_now_add=True)
-    completion_time = models.TimeField(auto_now_add=True)
+class Languages(models.Model):
+    id = models.BigIntegerField(primary_key=True)
+    created = models.DateTimeField(auto_now_add=True)
+    modified = models.DateTimeField(auto_now_add=True)
+    name = models.CharField()
+    abreviation = models.CharField()
+    prompt = models.CharField()
 
-    LANGUAGE_CHOICES = [
-        ('ES', 'Spanish'),
-        ('ZH', 'Traditional Chinese/Simplified Chinese'),
-        ('VI', 'Vietnamese'),
-        ('EN', 'English'),
-        ('UK', 'Ukrainian'),
-        ('RU', 'Russian'),
-        ('AR', 'Arabic'),
-        ('HT', 'Haitian Creole'),
-        ('FA', 'Persian'),
-        ('LO', 'Lao'),
-    ]
-    language = models.CharField(max_length=2, choices=LANGUAGE_CHOICES)
-
-    family_size = models.PositiveIntegerField()
-
-    SNAP_BENEFITS_CHOICES = [
-        ('YES', 'Yes'),
-        ('NO', 'No'),
-    ]
-    snap_benefits = models.CharField(max_length=3, choices=SNAP_BENEFITS_CHOICES)
-
-    TRAVEL_BY_CAR_CHOICES = [
-        ('YES', 'Yes'),
-        ('NO', 'No'),
-    ]
-    travel_by_car = models.CharField(max_length=3, choices=TRAVEL_BY_CAR_CHOICES)
-
-    zip_code = models.CharField(max_length=5)
+    def __str__(self):
+        return self.name
+    
+class TranslatedQuestions(models.Model):
+    created_timestamp = models.DateTimeField(auto_now_add=True)
+    modified_timestamp = models.DateTimeField(auto_now_add=True)
+    question_fk = models.ForeignKey("Questions", on_delete=models.SET_NULL, null=True)
+    language_fk = models.ForeignKey("Languages", on_delete=models.SET_NULL, null=True)
+    question = models.TextField()
+    answer_choices = ArrayField(models.CharField(blank=False))
+    '''
```

Comment:
We might need a string `other` field in this table to store the translation the word "other" if `has_other` for that question in `Questions` is `True`